ci-operator/step-registry/redhat-chaos/node-disruptions/master-outage/redhat-chaos-node-disruptions-master-outage-commands.sh

```diff
@@ -65,7 +65,24 @@ elif [ "$platform" = "IBMCloud" ]; then
     NODE_NAME=$(oc get nodes -l $LABEL_SELECTOR --no-headers | head -1 | awk '{printf $1}' )
     export NODE_NAME
     export TIMEOUT=320
-
+elif [ "$platform" = "VSphere" ]; then
+    export CLOUD_TYPE="vsphere"
+    # MASTER_MACHINE=$(oc get machines -A | grep master | head -n 1 | awk '{print $2}')
+    # VSPHERE_IP=$(oc get machine $MASTER_MACHINE -n openshift-machine-api -o jsonpath='{.status.addresses[?(@.type=="InternalIP")].address}' | awk '{print $1}')
+    # export VSPHERE_IP
+    VSPHERE_IP=$(oc get infrastructures.config.openshift.io cluster -o jsonpath='{.spec.platformSpec.vsphere.vcenters[0].server}')
+    export VSPHERE_IP
+    # VSPHERE_IP_WITHOUTDOT=$(echo "$VSPHERE_IP" | sed 's/\./\\./g')
+    # jsonpath_username="{.data.${VSPHERE_IP_WITHOUTDOT}\.username}"
+    # jsonpath_password="{.data.${VSPHERE_IP_WITHOUTDOT}\.password}"
+    # VSPHERE_USERNAME=$(oc get secret vsphere-creds -n kube-system -o jsonpath="$jsonpath_username")
+    # export VSPHERE_USERNAME
+    # VSPHERE_PASSWORD=$(oc get secret vsphere-creds -n kube-system -o jsonpath="$jsonpath_password")
+    # export VSPHERE_PASSWORD
+    vcenter_username_minimal_permission=$(cat "/var/run/vault/devqe-secrets/vcenter_username_minimal_permission")
```

Comment:
these keys won't be correct for the other credential name

I'm guessing it has a "secrets.sh" file with username/password. You'll want to source and see the variables created